docker/build.sh

```diff
@@ -1,4 +1,15 @@
-#!/bin/bash
+
+# Check if associative arrays arrays are supported
+unset assoc
+if ! declare -A assoc > /dev/null 2>&1 ; then
+    echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
```

Comment:
Please reduce this error to a one-liner. This is too much.

Reply:
I prefer if errors are clearly visible but thats a personal preference. I don't agree that it's too much but will change it.